spog/ui/README.md

```diff
@@ -8,13 +8,13 @@
 
 ```shell
 rustup target add wasm32-unknown-unknown
-cargo install trunk-ng
 cargo binstall trunk-ng # if you have `cargo-binstall` installed
+cargo install trunk-ng # or, otherwise
 ```
 
 > [!NOTE]
-> `trunk-ng` is a fork of `trunk`. Both should work at the moment, but `trunk-ng` has a bunch of fixes that `trunk` has
-> not.
+> `trunk-ng` is a fork of `trunk`. However, trunk-ng has additional features that we require. Trunk-ng version 0.17.16
+> or newer is required.
```

Comment:
I tested this PR just using trunk and not trunk-ng and everything worked as expected. Does it mean I'm just lucky or what exactly will work with trunk-ng that will not work with trunk?

Reply:
Not sure you clicked on the "scan sbom" link on the homepage.

The difference will be that bootstrap JS in the generated `index.html`, it will change the import to include all exported functions, and assign that to `window.wasmBindings`.

Reply:
Got it, you were right!